roles/mariadb/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# handlers file for mariadb
+
+- name: start mariadb server
```

Comment:
Uppercare. Before I review the rest of the changes, fix all this kind of mistakes. If you have so many problems with these details, look for an Ansible lint that will check them for you. I don't have a suggestion, but I'm sure you can find a lint.